packages/svelte/tests/runtime-runes/samples/derived-unowned-7/_config.js

```diff
@@ -9,6 +9,15 @@ export default test({
 			btn1.click();
 		});
 
-		assert.deepEqual(logs, ['computing C', 'computing B', 'a', 'foo', 'computing B', 'aaa', 'foo']);
+		assert.deepEqual(logs, [
+			'computing C',
+			'computing B',
+			'a',
+			'foo',
+			'computing B',
+			'aaa',
+			'computing C'
+			'foo'
```

Comment:
tbh this order is very confusing to me, the previous one as well TBH - could explain why it's running `b` before `c`? Is it because they're unowned and so the parent derived is executed eagerly / the dependency array of `c` is executed eagerly?

Reply:
Digging into it now

Reply:
Fixed.